core/schema/service.go

```diff
@@ -105,14 +123,87 @@ func (s *Service) Create(ctx context.Context, nsName string, schemaName string,
 		Compatibility: compatibility,
 	}
 	versionID := getIDforSchema(nsName, schemaName, sf.ID)
+	_, prevSchemaData, _ := s.GetLatest(ctx, nsName, schemaName)
```

Comment:
error is ignored! Please handle it.

Reply:
Maybe report it with log line and a metric and skip the ChangeIdentification, or else the next flows might break

Reply:
we can do metric once we have it implemented. Logged it for now.